packages/fx-core/src/plugins/solution/fx-solution/v2/deploy.ts

```diff
@@ -100,7 +100,7 @@ export async function deploy(
         )
       );
     }
-  } else if (envInfo.envName !== "local") {
+  } else if (envInfo.envName !== "local" && inputs[Constants.DEPLOY_AAD] !== "yes") {
```

Comment:
This is more than a message change. Any background on this one?

Reply:
This condition will hide select subscription if user hasn't been provisioned yet. Because aad related deploy do not need to select Azure related resource